frontend/src/pages/createWishlist/CreateWishlistPage.tsx

```diff
@@ -0,0 +1,12 @@
+import {Link} from "react-router-dom";
+import AddWishlist from "./AddWishlist";
+
+export default function CreateWishlistPage(){
+    return(
+        <div>
+
+            <AddWishlist></AddWishlist>
+
+        </div>
```

Comment:
Ich finds gut das du auslagerst!
In dem Fall ist es halt schön, wenn man die Komponenten auch etwas kleiner strickt (weil jetzt ist einfach ALLES in AddWishList drin und damit macht die Aufteilung keinen Sinn, aber wenn man dann aus AddWishList das AddWishListForm zB macht und den REst drumherum baut, erreicht man eine schöne Struktur mit kleinen, isolierten Komps)

Ist für jetzt aber erstmal OKAY